README.md

```diff
@@ -423,6 +422,82 @@ resp := &response{}
 err := c.RequestWithContext(ctx, "foo", req, resp)
 ```
 
+## JetStream Basic Usage
+
+```go
+import nats "github.com/nats-io/nats.go"
```

Comment:
Don't need nats here, Go does right thing. Your editor put that in but its not needed.

Reply:
Ah, ok. I'll update the existing example too then. 👍🏽